src/hotspot/share/prims/jvmtiEnvBase.hpp

```diff
@@ -382,7 +382,8 @@ class JvmtiEnvBase : public CHeapObj<mtInternal> {
   // get carrier thread last java vframe
   static javaVFrame* get_cthread_last_java_vframe(JavaThread* jt, RegisterMap* reg_map);
 
-  // get ordinary thread thread state
+  // get platform thread state
+  static jint get_thread_state_base(oop thread_oop, JavaThread* jt);
```

Comment:
maybe rename it to `get_platform_thread_state`?

Reply:
I was thinking about it. It will be inconsistent with`get_vthread_state`.
Ideally, then the `get_vthread_state` needs to be replaced with the `get_virtual_thread_state`.